doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
+
+For the light to take effect, you will need to enable environment lighting in the AppleseedOptions node:
+
+1. Select the AppleseedOptions node.
+
+2. In the _Node Editor_, expand the _Environment_ section.
+
+3. Toggle the switch next to the Environment Light plug to enable it.
+
+4. Type `/group/light` into the plug's field.
+
+The interactive render will now be in the process of updating, and you will be able to see Gaffy with some basic shaders and lighting.
+
+![The first render with a shader and lighting](images/viewerRenderOneShader.png "The first render with a shader and lighting")
+
+
+### Adding Textures ###
+
+As Gaffy is looking a bit bland, you should assign some textures to the shader:
+
+1. Create an Appleseed Color Texture node (_Appleseed_ > _Shader_ > _Texture2d_ > *As_color_texture*).
+
+2. In the _Node Editor_, point the node to the textures:
+	1. Type `${GAFFER_ROOT}/resources/gafferBot/textures/base_COL/base_COL_` into the _Filename_ value's field.
+	2. Select `mari` from the _UDIM_ drop-down menu.
+
+3. In the _Graph Editor_, connect the AppleSeed Color Texture node's ColorOut plug to the Disney Material node's BaseColor plug. Gaffy's textures will now mix with the shader, and the render will update to show the combined results.
+
+    ![Gaffy with textures](images/viewerRenderTextures.png "Gaffy with textures")
+
+
+### Adding Another Shader ###
+
+With the textures assigned, the surface of all of Gaffy's geometry looks the same, because the material shader is applying to everything. To fix this, you should create an additional metallic shader and apply it selectively to different parts of the geometry.
+
+Begin by creating another shader:
+
+1. Create another Disney Material node.
+
+2. In the _Node Editor_, give the shader some metallic properties:
+	1. Set Metallic to `0.8`
+	2. Set Roughness to `0.4`
+
+3. With the new Disney Material node still selected, create another ShaderAssignment node
+
+4. Click and drag the new ShaderAssignment node onto the connector under the first ShaderAssignment node. The new ShaderAssignment node will come after the fist shader in the graph flow.
+
+    ![A second shader in the Graph Editor](images/graphEditorSecondShaderNodes.png "A second shader in the Graph Editor")
+
+The _Viewer_ will update to show the new shader.
+
+![The second shader, rendered](images/mainRenderTwoShaders.png "The second shader, rendered")
+
+You will immediately notice something is wrong: _all_ the geometry is metallic. The new shader has overridden the previous one. This is because:
+
+> Important :
+> The last ShaderAssignment node applied to geometry in a scene takes precedence over all others.
+
+### Filtering a Shader ###
+
+In order to selectively apply a shader to only certain parts of the scene's geometry, you will need to **filter** the shader assignment, using a filter node that selects parts of a scene by name:
```

Comment:
"only certain _**locations in the**_ scene"

Reply:
lets skip " by name" since all filter nodes will work, and " by name" only pertains to the PathFilter